song_rater/tests.py

```diff
@@ -7,3 +8,19 @@ def test_home_page(self):
         response = c.get('/home')
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, 'Welcome to the homepage of Ordina Music Rater')
+
+
+class AddSongTestCase(TestCase):
+    def setUp(self):
+        Song.objects.create(artist='Apple', title='pie')
+
+    def test_add_song(self):
+        song = Song.objects.get(artist='Apple')
+        self.assertEqual(song.title, 'pie')
+
```

Comment:
Ik zou graag nog een test zien waar je een Post doet  en je verifieert dat de data die je hebt toegevoegd ook daadwerkelijk wordt opgeslagen in de DB.  Als je dan denkt "hoe dan?" laten we dan even pair programmen. 

Je kunt ook nog testen dat je het formulier opnieuw te zien krijgt als je data niet compleet is. Immers dit is logica die we zelf bedacht hebben (views.py regel 15)

Reply:
Implemented the suggested tests. Let's do pair programming if you have more suggestions concerning the tests